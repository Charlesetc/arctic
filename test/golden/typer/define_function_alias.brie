[
  define f f
  f a
]
