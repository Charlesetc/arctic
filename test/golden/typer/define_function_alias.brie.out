{[0, 1, 2, 5]=>Open_function(takes=3, returns=4)}